### ISSUE TITLE
Improve the visual differentiation of spell preparation modes for improved accessibility.

### ISSUE THREAD

**Issue by aaclayton** [foundryvtt/dnd5e#1281]
Originally in GitLab by @DrYoshiyahu

The icons for a spell being "prepared" and "always prepared" have such little contrast between them that they are almost completely unreadable at a glance, even for a person with near 20/20 vision, and they are *definitely* failing to meet any kind of [accessibility guidelines for contrasting colours](https://contrastchecker.online/44191a/4b4a44).

People with any kind of vision impairment would likely not even realise there are two different colours even on close inspection. At a minimum, the two symbols should be much much more different colours, but even better, they should be different symbols entirely.

![image](https://gitlab.com/foundrynet/dnd5e/uploads/f334ac21f952b08a4c7fbea215f0c115/image.png)

**Comment by Fyorl** [comment]
We now use a different icon entirely for 'always prepared', per #2992
